### PR TITLE
KAFKA-29: Set up CI (Gradle tasks)

### PR DESCRIPTION
**Pull request by paulgrape**

### CODE REVIEW

**Comment by paulgrape** [comment]
Almost without any changes
All Gradle tasks were checked

File: build.gradle [hunk 1]
```diff
@@ -26,6 +26,8 @@ group = "com.github.emc-mongoose"
 version = "4.2.6"
 sourceCompatibility = 11
 targetCompatibility = 11
+def mainClassName = "com.emc.mongoose.storage.driver.kafka.KafkaStorageDriver"
```
Comment (akurilov) [inline build.gradle]: not needed

File: build.gradle [hunk 2]
```diff
@@ -136,12 +139,13 @@ jar {
 			"Automatic-Module-Name": rootPkg,
 			"Implementation-Title": project.name,
 			"Implementation-Version": project.version,
+				"Main-Class": "$mainClassName",
```
Comment (akurilov) [inline build.gradle]: not needed

**Review by akurilov — DISMISSED** [review]

File: build.gradle (hunk 2 above)
Comment (paulgrape) [reply build.gradle]: Removed

File: build.gradle (hunk 1 above)
Comment (paulgrape) [reply build.gradle]: Removed

**Review by akurilov — APPROVED** [review]

**Pull request merged**